lambda/src/index.js

```diff
@@ -170,6 +171,16 @@ async function updateRunner (deploymentId, combinedRunner, event, error) {
   }
 }
 
+function generateEnvVars() {
+  const envVarsArray = []
+  const parsedVars = JSON.parse(Buffer.from(process.env.TEST_ENV_VAR_OVERRIDES, 'base64'))
+  if (Object.keys(parsedVars).length === 0) return envVarsArray
```

Comment:
I take it by this part, if nothing is passed in as env vars, it won't override anything (ie it won't be a breaking change). Is that right?

Reply:
Yes